### PR TITLE
Cry-O Feedback Tweaks

### PR DESCRIPTION
**Pull request by dustylens**
## About the PR

- Removes redundant carbon and oxygen from Salicylic Acid
- Adjusts Aloxadone caustic heal from 1 to 3
- Adds additional Aloxadone recipe replacing Sig with Siderlac.

## Why / Balance

Ongoing feedback regarding cryo medicine and the bounty system has routinely settled on the challenges of sustaining interest in attempting to recover bodies with certain types of damage due to the return not justifying the resources. The returns seem to be in a good place to me and the feedback has been consistent enough to warrant investigation.

- Salicylic Acid is a homebrew precursor that appears to be weighted down by redundant carbon and oxygen needs. Oxygen and carbon are factored into two other components of the recipe, in one place as a core ingredient, and do not impact the end yield. Removing them streamlines the recipe and better aligns the end result with expectations for input. If this is considered undesirable I would suggest adjusting the yield to 5 over 3.

- Aloxadone caustic restoration has been increased from 1 to 3. This was a major focus point for addressing damages of this type as the drain on resources seemed unnecessarily high. This brings the healing in line with the other damage types addressed. Which seems appropriate given that the primary medicines involved directly address caustic damage.

- Aloxadone receives a secondary recipe. Switching SIG for Siderlac. Siderlac is a botanical compound derived from Galaxy Thistle and Aloe, both of which are found in the medicine seed crate. Siderlac heals caustic damage, as does SIG. I believe that this will allow for chemists to create initial batches of Aloxadone from materials on hand and then swap over to trickling in sustaining quantities of meds as the shift progresses if more direct methods are being bottlenecked by resources.

## How to test

Check the recipes. Check that they're reflected in the guidebook. Give them a test.

## Requirements

- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

Nada

**Changelog**

🆑 
- tweak: Slight adjustments to Salicylic Acid recipe, additional recipe for Aloxadone and 2 point buff to Aloxadone caustic damage heal.

### CODE REVIEW

**Comment by noldevin** [comment]
I think this change would make Aloxadone more viable. Currently, I never bother to make it as it's not very commonly needed but player patients and sigy is a pain to make.

**Comment by AlekFenrir** [comment]
I think it'll do well, Allox is great to use, but a grind to get to and sometimes in the quantity required to heal someone it can be daunting for some.

**Review by whatston3 — COMMENTED** [review]
Caustic healing increase to Aloxadone seems fine, there are easier sources of caustic healing for live patients, and for patients with too much damage, fair.

Salicylic acid changes I think are unneeded, would prefer to have it work as a topical with effects similar to its entry in the TG chem wiki - you mentioned a desire for more advanced topicals, here's one.

Aloxadone recipe I think should be less reliant on botanicals.  Ditch the aloe, replace it with dermaline or its derivatives.

File: Resources/Prototypes/_NF/Recipes/Reactions/chemicals.yml [hunk 1]
```diff
@@ -5,10 +5,6 @@
       amount: 1
     Sodium:
       amount: 1
-    Carbon:
-      amount: 1
-    Oxygen:
-      amount: 1
```
[the remark this answers is not in the document]
Comment (whatston3) [reply Resources/Prototypes/_NF/Recipes/Reactions/chemicals.yml:11]: If the plan is to do something more with it as a niche, effective blanket anti-brute topical vs. the specialized bicaridine derivatives (again, reasonable to me), then I'd leave it at 3.

**Review by whatston3 — APPROVED** [review]
Seems reasonable to me.

Will note that salicylic acid change for later.

**Comment by chrome-cirrus** [comment]
Just want to say I really appreciate going the additional/alternate recipes path instead of replacing recipes (even if they're on paper simpler). It's nice to have multiple ways to get to an end and on-the-side use cases for when you end up with excesses of certain resources.

**Pull request merged**